core/network/PacketBuilder.hpp

```
@@ -346,4 +346,24 @@ struct PacketBuilder {
     }
+
+    static InputPlayerPacket makeInputPlayer(MovementInputType input, int sequence_number) {
+      InputPlayerPacket packet{};
+      packet.header.type = PacketType::InputPlayer;
+      packet.header.size = sizeof(packet);
+      packet.input = input;
+      packet.sequence_number = sequence_number;
+      return packet;
+    }
+
+    static PositionPlayerPacket makePositionPlayer(uint32_t player_id, float x, float y, std::uint32_t sequence_number) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)